cli/shared/src/test/scala/com/lightbend/rp/reactivecli/runtime/kubernetes/IngressJsonTest.scala

```diff
@@ -26,7 +26,7 @@ import utest._
 import Argonaut._
 
 object IngressJsonTest extends TestSuite {
-  def createAnnotations(appName: String, urlOne: String, urlTwo: String) =
+  def createAnnotations(appName: String, urlOne: String, urlTwo: String): Annotations =
```

Comment:
Scalac wouldn't compile without this. Inference started failing after I added a default value to the `Annotations` case class constructor.